test/integration/components/form.spec.ts

```diff
@@ -40,16 +40,14 @@ test('submits form data', async ({ page }) => {
       }
       return null;
     })
-    .toStrictEqual({
+    .toMatchObject({
       name: 'Toolpad',
       date: '1990-01-01',
       option: 'option 2',
       file: [
         {
           name: 'test.txt',
           type: 'text/plain',
-          size: 6,
```

Comment:
If this varies per platform you should also be able to do

```tsx
size: expect.any(Number),
base64: expect.stringMatching(/^data:text\/plain;base64,.+$/),
```
within `toMatchObject `

Reply:
I thought the file name and type were most important anyway, but yeah for some reason the size differs in Windows and the base64 URL was different and had `==` at the end, not sure why, I'll figure it out.